src/main/java/org/folio/circulation/domain/validation/ServicePointOfCheckoutPresentValidator.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.HttpResult.succeeded;
+
+import java.util.function.Function;
+
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.support.HttpResult;
+import org.folio.circulation.support.ValidationErrorFailure;
+
+public class ServicePointOfCheckoutPresentValidator {
+	private final Function<String, ValidationErrorFailure> servicePointOfCheckoutPresentFuntion;
+
+  public ServicePointOfCheckoutPresentValidator(
+			Function<String, ValidationErrorFailure> servicePointOfCheckoutPresentErrorFuntion) {
+
+		this.servicePointOfCheckoutPresentFuntion = servicePointOfCheckoutPresentErrorFuntion;
+  }
+
+	public HttpResult<LoanAndRelatedRecords> refuseServicePointOfCheckoutIsNotPresent(
```

Comment:
I think the intent is clearer if this is renamed ```suggestion
	public HttpResult<LoanAndRelatedRecords> refuseCheckOutWhenServicePointIsNotPresent(
```